### PR TITLE
[ADD] saas_sysadmin_route53_healthcheck

### PR DESCRIPTION
**Pull request by tarzan0820**
Enable route53 healthchecks for client databases

### CODE REVIEW

File: saas_sysadmin_route53_healthcheck/models/saas_sysdamin_route53_healthcheck.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+import logging
+from boto.route53.zone import Zone
```
Comment (yelizariev) [inline saas_sysadmin_route53_healthcheck/models/saas_sysdamin_route53_healthcheck.py:3]: move this to try block too

**Comment by yelizariev** [comment]
check comments

File: saas_sysadmin_route53_healthcheck/models/saas_sysdamin_route53_healthcheck.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+_logger = logging.getLogger(__name__)
+
+try:
+    import boto
+    from boto.route53.healthcheck import HealthCheck
+except:
+    _logger.critical('SAAS Route53 Requires the python library Boto which is not \
```
Comment (yelizariev) [inline saas_sysadmin_route53_healthcheck/models/saas_sysdamin_route53_healthcheck.py:10]: use _logger.debug instead

**Comment by yelizariev** [comment]
@tarzan0820 
could make small updates I mentioned?
Basically, I want to try runbot for this pull request (I have made it finally!)

Have you deleted your branch? Because PR shows "from `unknown repository`"

**Comment by saltonmassally** [comment]
@yelizariev congratulations on getting runbot up and running 👍 

I removed it, sorry I forgot to close the PR or mention it but I didn't think it was no longer necessary especially given that have several health checks on client instances swarmed the servers with request and these increased as the number of client instances did. It made more sense to just configure a single health check per server  (saas_server) and since these are relatively fixed and not variable like client instance, manual configure the healthchecks...

Sorry about not saying something about this but since I was the only one using route 53 I didn't think it would affect anyone else; however I still have the code on my local repo and can redo the PR if necessary

**Comment by yelizariev** [comment]
ok, let's close this PR.
As I understand you didn't have fixed version with health checking per saas server. Then, could you upload version you already have and share the link in #306

By the way, we start use route 53 for mailgun configuration.

**Pull request closed without merging**